app/components/copy-button.hbs

```diff
@@ -1,3 +1,4 @@
 <button type="button" ...attributes {{on "click" this.copy}}>
   {{yield}}
+  <EmberTooltip @text={{@hoverText}}/>
```

Comment:
I don't think the tooltip belongs in the `CopyButton` component itself. this component should only provide the "click-to-copy" functionality, but not any styling. if a button wants to use a tooltip then the `{{yield}}` should make it quite easy to add without mutating the `CopyButton` component itself.